build.sbt

```diff
@@ -358,7 +359,7 @@ lazy val nativelib =
     .settings(mavenPublishSettings)
     .settings(
       libraryDependencies += "org.scala-lang" % "scala-reflect" % scalaVersion.value,
-      publishLocal := publishLocal
+      update := update
```

Comment:
What happens to the `publishLocal` dependency in this case?

Reply:
Please keep me honest, correct, & proper. By my understanding, `nativelib / publishLocal`
will have gone through its earlier phases at some point. In particular, `nativelib / update`,
so that the spooky hidden nscplugin dependency I/we are chasing gets satisfied. 

Agreed entirely that this is not a good solution. The question before the house, IMHO,
is the benefit of the way faster fetch, especially over a slow wire,  make it an
acceptable fix whilst we figure out something better.  I propose "Yes".

Reply:
Using `publishLocal` like that says that another `publishLocal` must occur as a dependency when updating `nativelib` via `nativelib/publishLocal`. To `publishLocal` you must compile etc. and other project `dependsOn` must occur to. The top of the chain is `scalalib/publishLocal` from the rebuild alias.

Overall, I am just uncomfortable with changes other than the pulling of the library source as a basic replacement. I would have to test drive and look at outputs etc. Remember that if you break the chain then when working on the `javalib` for example, the process is to `javalib/publishLocal` after changes so you could be breaking some path.